### DISCUSSION TITLE
Can't go back to index page on github servers but on local i can

### DISCUSSION THREAD

**Discussion by lidormaymon** [community/community#52255] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

So i have these files which im uploading on my repository:
![a12bf87b21431af7393ca859844df6c1](https://user-images.githubusercontent.com/127669520/230608122-6f1a4a6e-fb9e-48db-b34d-7705139b8e0c.png)
Each folder is a different page what im trying to do is to navigate back to the index page via <a href>
This is the line i typed for it:
![1](https://user-images.githubusercontent.com/127669520/230608347-b8af96a5-ee21-43d2-bd82-07e83ea24c14.png)
When i press on the go back on localhost server it works fine as i want it to but when i do it on github server it shows me this:
![2](https://user-images.githubusercontent.com/127669520/230608481-600af1e5-9719-4813-9848-50b76b67c973.png)

**Comment by BREN0sx** [comment]
Hey! You must add your repository name in the path. 
I.e. use `/homework3/index.html`

Hope this helps! 😉

**Comment by lidormaymon** [comment]
woops thanks :)

**Reply by BREN0sx** [reply]
Right! Excellent 👊
